docs/build-instructions/linux.md

```diff
@@ -24,7 +24,7 @@ Ubuntu LTS 12.04 64-bit is the recommended platform.
 
 ### Fedora / CentOS / RHEL
 
-* `sudo yum --assumeyes install make gcc gcc-c++ glibc-devel git-core libgnome-keyring-devel rpmdevtools`
+* `sudo dnf --assumeyes install make gcc gcc-c++ glibc-devel git-core libgnome-keyring-devel rpmdevtools`
```

Comment:
This is qualified in the main readme as a fedora 21 vs. fedora 22+

https://github.com/atom/atom#red-hat-linux-fedora-21-and-under-centos-red-hat

Maybe we should do the same here?